src/dao/personDao.js

```diff
@@ -1,28 +1,50 @@
-var validation = require('../utils/validations');
-var personDB = require('../model/personModel.js');
+var personDB = require('../model/personModel');
 
-/*Validation constructor */
-function PersonDao() {
-}
-
-/*REturn messages by listMessagesId*/
-PersonDao.prototype.updateStateById = function (personId) {
+class PersonDao {
+    constructor() { }
+    
+    getPersonByEmail(personEmail) {
+        const getPersonByEmailPromise = new Promise((resolve, reject) => {
+            personDB.findOne({ 'email': personEmail }, function (err, personFound) {
```

Comment:
We are mixing arrow functions and anonymous functions, choose one (i prefer arrow function is the new js standard)

Reply:
Fixed